python/pyspark/pandas/series.py

```diff
@@ -3611,9 +3612,14 @@ def _rank(
             raise NotImplementedError("rank do not support MultiIndex now")
 
         if ascending:
-            asc_func = lambda scol: scol.asc()
+
+            def asc_func(scol: Column) -> Column:
+                return scol.asc()
+
         else:
-            asc_func = lambda scol: scol.desc()
+
+            def asc_func(scol: Column) -> Column:
+                return scol.desc()
```

Comment:
These two could be replaced with `sql.functions.{asc, desc}` or unapplied methods (`Column.asc` / `Column.desc`), couldn't they?